doc/ram.md

```diff
@@ -0,0 +1,36 @@
+# RAM
+
+To enable verification, the ledger will retain and modify, throughout
```

Comment:
```suggestion
To enable verification, the ledger will retain following data in its Non-volatile memory. It will also modify data after each signing operation,
```